python/xoscar/tests/core.py

```diff
@@ -53,6 +53,14 @@ def require_unix(func):
     return func
 
 
+def require_lunix(func):
+    if pytest:
+        func = pytest.mark.unix(func)
+
```

Comment:
Typo: require_linux and pytest.mark.linux(func)

Reply:
Done